docs/examples/promise/02_listen_to_blocks/exampleApiRx.js

```diff
@@ -0,0 +1,15 @@
+// Import the API Rx
```

Comment:
This is not promise - rxjs should go into /example/rxjs. Revert the naming changes.

Each sample is a runnable, installable file. Look at package.json, you can go to samples, do `yarn` and then `yarn start`